src/parser/ValueParser.ts

```diff
@@ -276,6 +283,28 @@ export class ValueParser {
       };
     });
 
+    if (literalValues.length > 0) {
```

Comment:
Does it handle nullable?

Reply:
No, I'm assuming that when there are literalValues, only parts of literalValues are needed

Reply:
I think it is expected that `'foo' | 'bar' | null` be generated to `SomeEnum?`. For reference, unions like `{ a: 1, b: 2 } | null` would be properly treated as optional values. Can you align with logic of how the tuple type is parsed a few lines below?

Reply:
I have added support for nullable.